helm/botkube/e2e-test-values.yaml

```diff
@@ -21,6 +24,30 @@ communications:
               - kubectl-read-only
             sources:
               - k8s-updates
+    discord:
+      enabled: false # Tests will override this temporarily
+      token: "" # Provide a valid token for BotKube app
+      botID: "" # Provide a valid Application Client ID for BotKube app
```

Comment:
Any reason to say `botID` instead of `clientID` or `appClientID`?

Reply:
That's how it's already configured in values.yaml:
```yaml
      # -- BotKube Application Client ID.
      botID: 'DISCORD_BOT_ID'
```